### ISSUE TITLE
CLI problem with progress complete's message

### ISSUE THREAD

**Issue by mixinix** [bcit-ci/CodeIgniter4#135]
CLI Progress, when it done shows message with repeated two characters from the end.

```
[..........] 0% Completete

Completete => .........tete
```

If I testing with changing the core (system/CLI/CLI.php):

```
Done => Donene (....nene)
```

looks better if I added two spaces (line 536):

```
fwrite(STDOUT, " {$percent}% Complete  ".PHP_EOL);
```

OS: Win8.1-x64 (console)

**Comment by jim-parry** [comment]
Bugfix Checklist:

- [ ] Component(s) modified
- [ ] Unit testing updated
- [ ] User guide updated
- [ ] Securely signed commits
- [ ] Conforms to style guide

**Comment by davidgv88** [comment]
Hi!. How can reproduce this bug?

**Comment by dlaynes** [comment]
Will the third parameter fix the issue? (Sorry for not testing this on my own)
fwrite($out, $str, strlen($str));

**Comment by lonnieezell** [comment]
Fixed in #334
